integration_tests/test_snap.py

```diff
@@ -81,8 +81,15 @@ def test_snap_default(self):
         self.assertThat(snap_file_path, FileExists())
 
     def test_cleanbuild(self):
-        self.skipTest("Fails to run correctly on travis.")
-        self.run_snapcraft('cleanbuild', 'assemble')
+        subprocess.check_call(['sudo', 'apt', 'install', '--yes', 'lxd'])
+        subprocess.check_call(['sudo', 'lxd', 'init', '--auto'])
```

Comment:
Re-iterating my question from before the latest changes, can we print /var/log/lxd/lxd.log here if this fails? Or consider using --logfile to /dev/stderr instead.